### PR TITLE
Interpretaties hydrogeologische stratigrafie

### PR DESCRIPTION
**Pull request by pjhaest**
* [x] I have read and followed the guidelines in the `CONTRIBUTING` document
* [x] I have checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change
* [x] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] All new and existing tests passed.

Describe your fixes/additions/changes
Search object for interpretations of hydrogeologische stratigrafie

### CODE REVIEW

File: pydov/search/boring.py [hunk 1]
```diff
@@ -113,3 +113,27 @@ def search(self, location=None, query=None, return_fields=None):
         df = pd.DataFrame(data=Boring.to_df_array(boringen, return_fields),
                           columns=Boring.get_field_names(return_fields))
         return df
+
```
Comment (Roel) [inline pydov/search/boring.py]: I think this can be removed.

File: pydov/search/interpretaties.py [hunk 2]
```diff
@@ -119,3 +120,142 @@ def search(self, location=None, query=None, return_fields=None):
...
+            fts, self.__wfs_namespace)
+
+        df = pd.DataFrame(
+            data=HydrogeologischeStratigrafie.to_df_array(
+                interpretaties, return_fields),
+            columns=HydrogeologischeStratigrafie.get_field_names(return_fields))
+        return df
+
+
+if __name__ == '__main__':
```
Comment (Roel) [inline pydov/search/interpretaties.py]: I think this can be removed.

File: pydov/search/interpretaties.py [hunk 3]
```diff
@@ -119,3 +120,142 @@ def search(self, location=None, query=None, return_fields=None):
...
+                        'type': field['type'],
+                        'wfs_injected': True
+                    })
+
+            self._fields = self._build_fields(
+                HydrogeologischeStratigrafieSearch.__wfs_schema,
+                HydrogeologischeStratigrafieSearch.__fc_featurecatalogue)
+
+    def search(self, location=None, query=None, return_fields=None):
+        """Search for boreholes (Boring). Provide either `location` or `query`.
```
Comment (Roel) [inline pydov/search/interpretaties.py]: This should be 'Hydrogeologische stratigrafie' instead of Boring.

File: pydov/types/interpretaties.py [hunk 4]
```diff
@@ -204,3 +204,198 @@ def _parse_xml_data(self):
...
+            Permanent key of the Boring (borehole), being a URI of the form
+            `https://www.dov.vlaanderen.be/data/boring/<id>`.
+
+        """
+        super(HydrogeologischeStratigrafie, self).__init__(
+            'interpretatie', pkey)
+
+    @classmethod
+    def from_wfs_element(cls, feature, namespace):
+        """Build `Boring` instance from a WFS feature element.
```
Comment (Roel) [inline pydov/types/interpretaties.py]: Should be HydrogeologischeStratigrafie

File: pydov/types/interpretaties.py [hunk 5]
```diff
@@ -204,3 +204,198 @@ def _parse_xml_data(self):
...
+        'type': 'float'
+    }]
+
+    def __init__(self, pkey):
+        """Initialisation.
+
+        Parameters
+        ----------
+        pkey : str
+            Permanent key of the Boring (borehole), being a URI of the form
```
Comment (Roel) [inline pydov/types/interpretaties.py]: Should be HydrogeologischeStratigrafie

File: pydov/types/interpretaties.py [hunk 6]
```diff
@@ -204,3 +204,198 @@ def _parse_xml_data(self):
...
+        Parameters
+        ----------
+        feature : etree.Element
+            XML element representing a single record of the WFS layer.
+        namespace : str
+            Namespace associated with this WFS featuretype.
+
+        Returns
+        -------
+        boring : Boring
```
Comment (Roel) [inline pydov/types/interpretaties.py]: Should be HydrogeologischeStratigrafie

Comment (pjhaest) [reply pydov/types/interpretaties.py]: Ok, thx for the review, changed all docstrings in new commit

**Pull request merged**